src/backend/tests/tasks.test.ts

```diff
@@ -202,4 +203,70 @@ describe('Tasks', () => {
       );
     });
   });
+
+  describe('editTaskAssignees', () => {
+    test('edit task assignee succeeds', async () => {
+      jest.spyOn(prisma.task, 'findUnique').mockResolvedValue(taskSaveTheDayPrisma);
+      jest.spyOn(prisma.task, 'update').mockResolvedValue(taskSaveTheDayInProgressPrisma);
+      jest.spyOn(taskTransformer, 'default').mockReturnValue(taskSaveTheDayInProgressShared);
+      jest.spyOn(userUtils, 'getUsers').mockResolvedValue([superman, wonderwoman]);
+
+      const taskId = '1';
+      const userIds = [
+        {
+          userId: 2
```

Comment:
this object should probably be `[{ userId: superman.userId }, { userId: wonderwoman.userId }]` to avoid hardcoding problems if the data ever changes